### PR TITLE
Temp/ah/mcu json

### PR DESCRIPTION
**Pull request by ariel-hi**
===== Microcontrollers in OCDB =====
Added new stm land patterns.
Rename `to-esir` to `to-jitx`.
Connect the new STM JSON to `micro-controllers.stanza`.
Generate JITX pin-properties, bundles, and supports dynamically.
Includes multiple test JSON files in the `utils` folder. These will be shuffled around.

### CODE REVIEW

**Comment by PhilippeFerreiraDeSousa** [comment]
Did you commit json files by mistake or are there tests that use them as data source?

**Comment by RajeshJITX** [comment]
> Did you commit json files by mistake or are there tests that use them as data source?

We won't need these files now that we have mongodb. We can run tests against mongodb. I can also see value in having tests files right there so we can run unit tests without any network connection

File: utils/micro-controllers.stanza [hunk 1]
```diff
@@ -186,6 +127,91 @@ public defn MicroController (properties:Tuple<KeyValue>, exist:Tuple<String>, so
...
+    mpn = mpn(mcu)
+
+    to-jitx-pin-properties(pin-properties(mcu))
+    val bundle-table = to-jitx-bundles(bundles(mcu))
+    to-jitx-supports(supports(mcu), bundle-table)
+
+    assign-landpattern $
+      switch(mfg-package(mcu)) :
+        ; "SO8N" : SO8N
+        ; "TSSOP14" : TSSOP14
```
Comment (RajeshJITX) [inline utils/micro-controllers.stanza:146]: Why are these commented? Is it just for testing or do we need to implement these land patterns?

**Review by RajeshJITX — APPROVED** [review]
Looks good.

File: utils/micro-controllers.stanza (hunk 1 above)
Comment (ariel-hi) [reply utils/micro-controllers.stanza:146]: The commented ones are land patterns for which we haven't yet coded an internal definition. Yep, to be implemented.
Our queries need a way to prioritize land patterns that are implemented.

**Comment by ariel-hi** [comment]
Closing in favor of new draft PR.

**Pull request closed without merging**